db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -2505,6 +2506,9 @@ void DBImpl::BackgroundCallFlush(Env::Priority thread_pri) {
     InstrumentedMutexLock l(&mutex_);
     assert(bg_flush_scheduled_);
     num_running_flushes_++;
+    if (thread_pri == Env::Priority::LOW) {
+      bg_flush_scheduled_low_--;
```

Comment:
I think better decrease bg_flush_scheduled_low_ after flush is really completed, like right after `bg_flush_scheduled_--` at line 2568

Reply:
Not sure if we even need ```bg_flush_scheduled_low_``` if we consider flushes and compactions together.